.github/workflows/nix-build-and-upload.yaml

```diff
@@ -0,0 +1,112 @@
+name: nix_p4-fusion
+
+on:
+  push:
+    branches: 
+      - main
+      - nsc/nix-build # temporary
+  # pull_request:
+
+concurrency:
+  group: ${{ github.ref }}-nix
+  cancel-in-progress: ${{ github.ref != 'refs/heads/master' }}
+
+jobs:
+  x86_64-darwin:
```

Comment:
🤮 x86 darwin, so old-school 😬

Reply:
I think theres a non-zero amount of people at sg with them 😬

Reply:
Let’s add PowerPC, just to be sure 😅

Reply:
You know what, it's a legitimate question to ask, we keep adding `darwmin-amd64` left and right, so I'd rather not have to if we don't have anyone left with those old macbooks.